db/migrations/202407201604_transactions_indexes.go

```diff
@@ -12,7 +12,7 @@ import (
 // - Adds indexes that should speed up queries related to transactions
 // - (basic indexes)
 // - index for budget query
-// - index for isolated balance queries
+// - index for isolated/sub-wallet balance queries
```

Comment:
```suggestion
// - index for isolated balance queries
```

No need to change this one as it is just a technical comment